helix-view/src/editor.rs

```diff
@@ -183,6 +183,11 @@ impl Editor {
         Ok(id)
     }
 
+    pub fn reload(&mut self) -> Result<(), Error> {
```

Comment:
I think it's okay to drop this and do it directly in the command. `editor.reload()` with no parameters looks more like you're reloading all the documents

Reply:
Fixed in https://github.com/helix-editor/helix/pull/177/commits/ba3468886172bf7354ca7211760f2451d9322a20.